src/components/404/404.css

```diff
@@ -0,0 +1,8 @@
+body {
```

Comment:
we might need to change this to a specific tag, we can only have one body in whole app css as they clash with each other even when you don't import other css files to your component js